R/stratify_data.R

```diff
@@ -1,5 +1,8 @@
+
+utils::globalVariables(c("value", "age"))
```

Comment:
please don't create global variables

Reply:
This is well-noted. I was having errors on these two variables not being found in the provided datasets. Initially, I thought it was because there not visible globally. I have so far ascertained the error was not on global variables but on naming of datasets. I will remove the line of code.